src/wallet/ismine.h

```diff
@@ -17,13 +17,13 @@ class CScript;
 /** IsMine() return codes */
 enum isminetype : unsigned int
 {
-    ISMINE_NO         = 0,
-    ISMINE_WATCH_ONLY = 1 << 0,
-    ISMINE_SPENDABLE  = 1 << 1,
-    ISMINE_USED       = 1 << 2,
+    ISMINE_NO         = 0, // the scriptPubKey is not contained in the wallet
```

Comment:
Please use `//<! ` here so that doxygen picks up the descriptions

Reply:
@laanwj    @fanquake 
fixed,  can u take a look again? thx~